rpc-proxy/rpc-proxy.cabal

```diff
@@ -38,7 +38,8 @@ Executable rpc-proxy
     transformers-base,
     xchwebsocket,
     xchxenstore,
-    xchdb
+    xchdb,
+    rpc-autogen
```

Comment:
I'm surprised you had to add rpc-autogen.  xchdb switched to rpc-autogen in its own commit, so I would think that is sufficient.  Does rpc-proxy need some other idl binding that it was receiving transitively?

I don't know cabal, so I may be totally wrong.

Reply:
rpcgen does have its dbus stub generated from an [xml](https://github.com/OpenXT/idl/blob/master/interfaces/rpc_proxy.xml#L6), which is why I put the explicit dependency, but it could get the object from xchdb. I can check that.